core/parser.mly

```diff
@@ -906,6 +906,7 @@ lens_expression:
 | LENS exp DEFAULT                                             { with_pos $loc (LensLit ($2, None))}
 | LENS exp TABLEKEYS exp                                       { with_pos $loc (LensKeysLit ($2, $4, None))}
 | LENS exp WITH LBRACE fn_deps RBRACE                          { with_pos $loc (LensFunDepsLit ($2, $5, None))}
+| LENSSERIAL exp WITH VARIABLE+                                               { with_pos $loc (LensSerialLit ($2, $4, None))}
```

Comment:
Been punching the spacebar too many times?

Reply:
Fixed this formatting error.